src/main/resources/db/createDb.sql

```diff
@@ -0,0 +1,22 @@
+create database statistic;
```

Comment:
createDB.sql - Must be moved out of resources directory. For example in root of the project because this script isn't used in your code.